<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Основные сведения</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="pathLineEdit">
                 <property name="placeholderText">
                  <string>Расположение файла</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="fileOpenButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="sizeLabel">
               <property name="text">
                <string>Размер файла:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="creationLabel">
               <property name="text">
                <string>Дата и время создания файла:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="modificationLabel">
               <property name="text">
                <string>Дата и время последнего изменения:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="packCountLabel">
               <property name="text">
                <string>Количество пакетов:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="maxLenLabel">
               <property name="text">
                <string>Максимальная длина пакета:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="minLenLabel">
               <property name="text">
                <string>Минимальная длина пакета:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Отображение пакетных структур</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="binaryButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="decButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>10</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="hexButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>16</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="textButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>T</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>70</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="sigViewer">
             <property name="minimumSize">
              <size>
               <width>460</width>
               <height>159</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Отображение найденных символов и кодов</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTextBrowser" name="textBrowser"/>
             </item>
             <item>
              <widget class="QProgressBar" name="decompressionProgressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="readedBitCountLabel">
                 <property name="text">
                  <string>Прочитано бит:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearTextBrowserButton">
                 <property name="maximumSize">
                  <size>
                   <width>101</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Очистить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Вывод алфавитов</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QGroupBox" name="charAlfBox">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>140</height>
                </size>
               </property>
               <property name="title">
                <string>Алфавит символов</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QProgressBar" name="charAlfProgress">
                    <property name="value">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="charAlfStatus">
                    <property name="text">
                     <string>Статус</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="charAlfCount">
                    <property name="text">
                     <string>Элементы: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="charAlfPresentButton">
                    <property name="text">
                     <string>Вывести</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="lenAlfBox">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>140</height>
                </size>
               </property>
               <property name="title">
                <string>Алфавит длин</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QProgressBar" name="lenAlfProgress">
                    <property name="value">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lenAlfStatus">
                    <property name="text">
                     <string>Статус</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lenAlfCount">
                    <property name="text">
                     <string>Элементы:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="lenAlfPresentButton">
                    <property name="text">
                     <string>Вывести</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="moveAlfBox">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>140</height>
                </size>
               </property>
               <property name="title">
                <string>Алфавит смещений</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QProgressBar" name="moveAlfProgress">
                    <property name="value">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="moveAlfStatus">
                    <property name="text">
                     <string>Статус</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="moveAlfCount">
                    <property name="text">
                     <string>Элементы:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="moveAlfPresentButton">
                    <property name="text">
                     <string>Вывести</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Парраметры обработки</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QRadioButton" name="decodeFlow">
                 <property name="text">
                  <string>Декодировать весь поток</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="decodeBlocs">
                 <property name="text">
                  <string>Декодировать по блокам</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="seekPositionLabel">
                   <property name="text">
                    <string>Позиция начального бита:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="seekSpinBox">
                   <property name="maximum">
                    <number>9999</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QCheckBox" name="staticBlocks">
                 <property name="text">
                  <string>Искать статические коды</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="dynamicBlocks">
                 <property name="text">
                  <string>Искать динамические коды</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="noCompressionBlocks">
                 <property name="text">
                  <string>Искать блоки без сжатия</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Декодировать</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="errorFixerButton">
      <property name="text">
       <string>Исправление ошибок</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
